.gitignore

```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 .publish
 npm-debug.log
+yarn.lock
```

Comment:
let's not gitignore this

Reply:
@ianks  we weren't tracking it before so do you want to start tracking it?